src/radical/pilot/agent/agent_0.py

```diff
@@ -83,7 +86,9 @@ def __init__(self, cfg, session):
         self._cmgr.start_components()
 
         # start any services if they are requested
-        self._start_services()
+        # TODO: Remove this. do not start services from here
```

Comment:
You inserted `start_services` below, so you can indeed remove these commented lines.

Reply:
yes